{
    "base-errors-activity_invite_already_invited": "The user __name__ is already invited for this activity"
}
